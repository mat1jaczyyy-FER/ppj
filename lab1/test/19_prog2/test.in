abc = 765+123/ (123
-123
+32)

za a od 3 do 5
  xy = abc+32 // nop
azabc = 765+123/ (123
-123
+32)

za a od 3 do 5
  xy = abc+32 // nop
az
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
