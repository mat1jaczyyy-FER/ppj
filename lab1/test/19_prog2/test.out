IDN 1 abc
OP_PRIDRUZI 1 =
BROJ 1 765
OP_PLUS 1 +
BROJ 1 123
OP_DIJELI 1 /
L_ZAGRADA 1 (
BROJ 1 123
OP_MINUS 2 -
BROJ 2 123
OP_PLUS 3 +
BROJ 3 32
D_ZAGRADA 3 )
KR_ZA 5 za
IDN 5 a
KR_OD 5 od
BROJ 5 3
KR_DO 5 do
BROJ 5 5
IDN 6 xy
OP_PRIDRUZI 6 =
IDN 6 abc
OP_PLUS 6 +
BROJ 6 32
IDN 7 azabc
OP_PRIDRUZI 7 =
BROJ 7 765
OP_PLUS 7 +
BROJ 7 123
OP_DIJELI 7 /
L_ZAGRADA 7 (
BROJ 7 123
OP_MINUS 8 -
BROJ 8 123
OP_PLUS 9 +
BROJ 9 32
D_ZAGRADA 9 )
KR_ZA 11 za
IDN 11 a
KR_OD 11 od
BROJ 11 3
KR_DO 11 do
BROJ 11 5
IDN 12 xy
OP_PRIDRUZI 12 =
IDN 12 abc
OP_PLUS 12 +
BROJ 12 32
KR_AZ 13 az
KR_ZA 14 za
IDN 14 a
KR_OD 14 od
BROJ 14 3
KR_DO 14 do
BROJ 14 5
KR_ZA 15 za
IDN 15 a
KR_OD 15 od
BROJ 15 3
KR_DO 15 do
BROJ 15 5
KR_ZA 16 za
IDN 16 a
KR_OD 16 od
BROJ 16 3
KR_DO 16 do
BROJ 16 5
KR_ZA 17 za
IDN 17 a
KR_OD 17 od
BROJ 17 3
KR_DO 17 do
BROJ 17 5
KR_ZA 18 za
IDN 18 a
KR_OD 18 od
BROJ 18 3
KR_DO 18 do
BROJ 18 5
KR_ZA 19 za
IDN 19 a
KR_OD 19 od
BROJ 19 3
KR_DO 19 do
BROJ 19 5
KR_ZA 20 za
IDN 20 a
KR_OD 20 od
BROJ 20 3
KR_DO 20 do
BROJ 20 5
KR_ZA 21 za
IDN 21 a
KR_OD 21 od
BROJ 21 3
KR_DO 21 do
BROJ 21 5
KR_ZA 22 za
IDN 22 a
KR_OD 22 od
BROJ 22 3
KR_DO 22 do
BROJ 22 5
KR_ZA 23 za
IDN 23 a
KR_OD 23 od
BROJ 23 3
KR_DO 23 do
BROJ 23 5
KR_ZA 24 za
IDN 24 a
KR_OD 24 od
BROJ 24 3
KR_DO 24 do
BROJ 24 5
KR_ZA 25 za
IDN 25 a
KR_OD 25 od
BROJ 25 3
KR_DO 25 do
BROJ 25 5
IDN 26 xy
OP_PRIDRUZI 26 =
IDN 26 abc
OP_PLUS 26 +
BROJ 26 32
KR_AZ 27 az
IDN 28 xy
OP_PRIDRUZI 28 =
IDN 28 abc
OP_PLUS 28 +
BROJ 28 32
KR_AZ 29 az
IDN 30 xy
OP_PRIDRUZI 30 =
IDN 30 abc
OP_PLUS 30 +
BROJ 30 32
KR_AZ 31 az
IDN 32 xy
OP_PRIDRUZI 32 =
IDN 32 abc
OP_PLUS 32 +
BROJ 32 32
KR_AZ 33 az
IDN 34 xy
OP_PRIDRUZI 34 =
IDN 34 abc
OP_PLUS 34 +
BROJ 34 32
KR_AZ 35 az
IDN 36 xy
OP_PRIDRUZI 36 =
IDN 36 abc
OP_PLUS 36 +
BROJ 36 32
KR_AZ 37 az
IDN 38 xy
OP_PRIDRUZI 38 =
IDN 38 abc
OP_PLUS 38 +
BROJ 38 32
KR_AZ 39 az
IDN 40 xy
OP_PRIDRUZI 40 =
IDN 40 abc
OP_PLUS 40 +
BROJ 40 32
KR_AZ 41 az
IDN 42 xy
OP_PRIDRUZI 42 =
IDN 42 abc
OP_PLUS 42 +
BROJ 42 32
KR_AZ 43 az
IDN 44 xy
OP_PRIDRUZI 44 =
IDN 44 abc
OP_PLUS 44 +
BROJ 44 32
KR_AZ 45 az
IDN 46 xy
OP_PRIDRUZI 46 =
IDN 46 abc
OP_PLUS 46 +
BROJ 46 32
KR_AZ 47 az
IDN 48 xy
OP_PRIDRUZI 48 =
IDN 48 abc
OP_PLUS 48 +
BROJ 48 32
KR_AZ 49 az
